internal/support/logging/init.go

```diff
@@ -39,13 +39,13 @@ func NewServiceInit(server server) ServiceInit {
 	}
 }
 
-func getPersistence() (persistence, error) {
+func getPersistence(dic *di.Container) (persistence, error) {
```

Comment:
Don't pass the dic around.  It's a god object intended to bridge the gap after bootstrap/init between routing and controller functions/methods.  Pull out required dependencies in the caller and pass those dependencies explicitly to getPersistence.